assets/javascripts/discourse/components/remind-assigns-frequency.js.es6

```diff
@@ -0,0 +1,23 @@
+import computed from "ember-addons/ember-computed-decorators";
+
+export default Ember.Component.extend({
+  @computed("user.reminders_frequency")
+  translatedFrequencies() {
```

Comment:
Something about the name feels odd to me because the property is actually building the content for the combobox. Translating the frequency is a subset of what this component does.